inlong-manager/manager-dao/src/main/resources/mappers/StreamSinkFieldEntityMapper.xml

```diff
@@ -37,10 +37,13 @@
         <result column="field_format" jdbcType="VARCHAR" property="fieldFormat"/>
         <result column="rank_num" jdbcType="SMALLINT" property="rankNum"/>
         <result column="is_deleted" jdbcType="INTEGER" property="isDeleted"/>
+        <result column="origin_node_name" jdbcType="VARCHAR" property="originNodeName"/>
```

Comment:
Suggests moving those two fields after the `field_format` field, because the `is_deleted` field was a unified field of each table, it should be at the end of each table.

Reply:
done